docs/Istio-on-karmada-different-network.md

```diff
@@ -0,0 +1,294 @@
+```
+
+add `topology.istio.io/network: network2` in `.spec.workload.manifests.metadata.labels`
+
+3、Install istio control plane
+
+Export `KUBECONFIG` and switch to `member1`:
+
+```bash
+export KUBECONFIG="$HOME/.kube/members.config"
+kubectl config use-context member1
+```
+
+```bash
+cat <<EOF | istioctl install -y -f -
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    accessLogFile: /dev/stdout
+  values:
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: member1
+      network: network1
+EOF
+```
+
+4、Install the east-west gateway in `member1`
+
+```bash
+samples/multicluster/gen-eastwest-gateway.sh --mesh mesh1 --cluster member1 --network network1 | istioctl install -y -f -
+```
+
+5、Expose the control plane and service in `member1`
+
+```bash
+kubectl apply -f samples/multicluster/expose-istiod.yaml -n istio-system
+kubectl apply -f samples/multicluster/expose-services.yaml -n istio-system
+```
+
+### Configure `member2` as a remote cluster
+
+1、Enable API ServerAccess to `member2`
+
+switch to `member2`:
+
+```bash
+kubectl config use-context member2
+```
+
+Prepare member2 cluster secret
+
+```bash
+istioctl x create-remote-secret --name=member2 > istio-remote-secret-member2.yaml
+```
+
+Switch to `member1`:
+
+```bash
+kubectl config use-context member1
+```
+
+Apply istio remote secret
+
+```bash
+kubectl apply -f istio-remote-secret-member2.yaml
+```
+
+2、configure member2 as a remote
+
+Save the address of `member1`’s east-west gateway
+
+```bash
+export DISCOVERY_ADDRESS=$(kubectl -n istio-system get svc istio-eastwestgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+```
+
+Create a remote configuration on `member2`.
+
+Switch to `member2`:
+
+```bash
+kubectl config use-context member2
+```
+
+```bash
+cat <<EOF | istioctl install -y -f -
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: member2
+      network: network2
+      remotePilotAddress: ${DISCOVERY_ADDRESS}
+EOF
+```
+
+3、Install the east-west gateway in `member2`
+
+```bash
+samples/multicluster/gen-eastwest-gateway.sh --mesh mesh1 --cluster member2 --network network2 | istioctl install -y -f -
+```
+
+4、Expose service in `member2`
+
+```bash
+kubectl apply -f samples/multicluster/expose-services.yaml -n istio-system
+```
+
+### Deploy bookinfo application
+
+1、Open Karmada's auto-sync feature
+
+Export `KUBECONFIG` and switch to `karmada host`:
+
+```bash
+export KUBECONFIG=$HOME/.kube/karmada.config
+kubectl config use-context karmada-host
+```
+
+Edit `karmada-controller-manager`deployment
+
+```bash
+kubectl edit deployment karmada-controller-manager -n karmada-system
+```
+
+delete `--controllers=-namespace,*` in start command
+
+2、Deploy bookinfo application
+
+See module `Deploy bookinfo application` in https://github.com/suwliang3/karmada/blob/master/docs/istio-on-karmada.md
```

Comment:
> See module `Deploy bookinfo application` in https://github.com/suwliang3/karmada/blob/master/docs/istio-on-karmada.md
Why not use [official documents](https://github.com/karmada-io/karmada/blob/master/docs/istio-on-karmada.md)?